### PR TITLE
whitney font

### PR DESCRIPTION
**Pull request by afred**
@mccalluc? more toward #25 to come... but here is smaller PR for review.

### CODE REVIEW

**Comment by afred** [comment]
### Screenshots
image

image

image

image

File: app/assets/stylesheets/globals/fonts.css.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
 // Whitney (sans serif):
-//@import url("http://cloud.typography.com/7419092/765888/css/fonts.css");
-// TODO?
\ No newline at end of file
+@import url("//cloud.typography.com/7419092/6220752/css/fonts.css");
+
+$font-family: "Whitney SSm A", "Whitney SSm B", sans-serif;
```
Comment (mccalluc) [inline app/assets/stylesheets/globals/fonts.css.scss]: Does this get used anywhere other than defaults.css.scss? If not, maybe just do the actual font-family declaration here? (Just worry about someone tweaking default.css at some point, but not cleaning this up, and then someone else tries to make a tweak here... all rather hypothetical.)

Comment (afred) [inline app/assets/stylesheets/globals/fonts.css.scss]: i was trying to keep all of the "Whitney"-related stuff in one file.

Comment (afred) [inline app/assets/stylesheets/globals/fonts.css.scss]: i like this: https://robots.thoughtbot.com/sass-default

new commit coming.. stand by.

**Comment by mccalluc** [comment]
@afred:
- I kind of liked the old, 2-column footer when narrow: We have the room, so why not use it?
- Also, in the original spec, the homepage footer is white, unlike the other pages. I think that helps with the grey box just above: grey-white is cleaner than grey-white-another-grey.

screen shot 2016-02-11 at 5 58 40 pm

but nothing I'd block on, so I'll go ahead and merge.

**Comment by afred** [comment]
homepage footer is not white in the spec: https://atlas.wgbh.org/confluence/display/OV/OV+Styled+Comps+V.4. I just used the light-grey that was defined. But.. i agree with you.. different shade of gray doesn't do much. I'll change it back next PR.

**Comment by mccalluc** [comment]
oh, you're right: the light-grey/dark-grey contrast is so much more than the grey/white that I didn't notice... partly because there's no background white between the two. (One thing I had trouble with in these specs is judging what's a banner that spreads the whole width, and what should only go to the margin.)

**Comment by afred** [comment]
@mccalluc yeah, I noticed that too. I guess design assumed a fixed width layout?

**Pull request merged**